### ISSUE TITLE
php artisan migrate keeps trying to access the wink DB connection

### ISSUE THREAD

**Issue by connecteev** [writingink/wink#130]
I have a Laravel app set up with Wink. Wink is able to pull from the wink database, but the main laravel db connection cannot be accessed anymore. Is this a bug in Wink, because it certainly looks that way?

This is from config/database.php:

        'mysql' => [
            'driver' => 'mysql',
            'host' => env('DB_HOST', '127.0.0.1'),
            'port' => env('DB_PORT', '3306'),
            'database' => env('DB_DATABASE', 'forge'),
            'username' => env('DB_USERNAME', 'forge'),
            'password' => env('DB_PASSWORD', ''),
            'unix_socket' => env('DB_SOCKET', ''),
            'charset' => 'utf8',
            'collation' => 'utf8_unicode_ci',
            'prefix' => '',
            'prefix_indexes' => true,
            'strict' => true,
            'engine' => null,
        ],
        'wink_connection' => [
            'driver' => 'mysql',
            'host' => env('DB_HOST', '127.0.0.1'),
            'port' => env('DB_PORT', '3306'),
            'database' => env('DB_WINK_DATABASE', 'forge'),
            'username' => env('DB_WINK_USERNAME', 'forge'),
            'password' => env('DB_WINK_PASSWORD', ''),
            'unix_socket' => env('DB_SOCKET', ''),
            'charset' => 'utf8',
            'collation' => 'utf8_unicode_ci',
            'prefix' => '',
            'prefix_indexes' => true,
            'strict' => true,
            'engine' => null,
        ]

This is in config/wink.php:
`return [
    'database_connection' => env('WINK_DB_CONNECTION', 'wink'),
    'storage_disk' => env('WINK_STORAGE_DISK', 'local'),
    'storage_path' => env('WINK_STORAGE_PATH', 'public/wink/images'),
    'path' => env('WINK_PATH', 'wink'),
`

And this is in my .env file:
`

DB_CONNECTION=mysql
DB_HOST=127.0.0.1
DB_PORT=3306
DB_DATABASE=kb_skins
DB_USERNAME=root
DB_PASSWORD=root
WINK_BLOG_PREFIX=learn
WINK_BLOG_TAG_URL=topic
WINK_DB_CONNECTION=wink_connection
DB_WINK_DATABASE=kb_wink
DB_WINK_USERNAME=root
DB_WINK_PASSWORD=root

`

Every time I try to run 
`php artisan migrate`

It keeps trying to access the wink db connection instead of the main Laravel db connection. I've cleared the app cache using
`
php artisan config:clear
php artisan cache:clear
php artisan route:clear
php artisan view:clear
php artisan optimize
`
restarted the laravel server, etc. Nothing works. Here's the error:

php artisan migrate 
`
Migrating: 2018_10_30_000000_create_tables

   Illuminate\Database\QueryException  : SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'wink_tags' already exists (SQL: create table `wink_tags` (`id` char(36) not null, `slug` varchar(255) not null, `name` varchar(255) not null, `created_at` timestamp null, `updated_at` timestamp null) default character set utf8 collate 'utf8_unicode_ci')

  at /Code/kb_skins/vendor/laravel/framework/src/Illuminate/Database/Connection.php:664
    660|         // If an exception occurs when attempting to run a query, we'll format the error
    661|         // message to include the bindings with SQL, which will make this exception a
    662|         // lot more helpful to the developer instead of just the database's errors.
    663|         catch (Exception $e) {
  > 664|             throw new QueryException(
    665|                 $query, $this->prepareBindings($bindings), $e
    666|             );
    667|         }
    668| 

  Exception trace:
  1   PDOException::("SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'wink_tags' already exists")
      /Code/kb_skins/vendor/laravel/framework/src/Illuminate/Database/Connection.php:458
  2   PDOStatement::execute()
      /Code/kb_skins/vendor/laravel/framework/src/Illuminate/Database/Connection.php:458
  Please use the argument -v to see more details.
`

**Comment by CraftLogan** [comment]
Hello, @connecteev 

I believe the problem might be the tables have already been migrated

You could try the command `php artisan migrate:refresh` and see if that works for you. Thanks Logan

**Comment by themsaid** [comment]
fixed here: https://github.com/writingink/wink/commit/6241562599fde3e7f6848e25d083592a0056db6e

Not Wink won't register migrations. You must run `wink:migrate` to run wink migrations.

**Comment by connecteev** [comment]
@themsaid thank you for the quick update on this! I'm learning a bit about laravel packages thanks to wink.

So, this is strange but it still doesnt work (/vendor/writingink/wink/src/WinkServiceProvider.php is not getting updated). I ran these and verified that my composer.json is updated:
update composer.json run:
composer require writingink/wink

`
    "require": {
        "php": "^7.1.3",
        "fideloper/proxy": "^4.0",
        "laravel/framework": "5.7.*",
        "laravel/tinker": "^1.0",
        "writingink/wink": "^0.0.11"
    },
`
And I also ran `php artisan migrate` after cleaning out the cache:
`
Clear Cache facade value: php artisan cache:clear.
Clear Route cache: php artisan route:cache.
Clear View cache: php artisan view:clear.
Clear Config cache: php artisan config:cache.
`

Am I missing a step or just doing something silly?
